src/python/traces/sampling/recipefile.json

```
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../../../otel-recipes-schema.json",
+  "id": "python.traces.sampling",
+  "languageId": "python",
+  "signal": "traces",
+  "displayName": "Sampling App",
```

Don't know if it's the best name we can use. Appreciate any suggestions

---

It can be a bit longer yeah. Maybe

```suggestion
  "displayName": "Console app using a TraceIdRatioBased sampler",
```

---

Fixed